experimental/regression_suite/shark-test-suite-models/sdxl/test_unet.py

```diff
@@ -61,6 +62,48 @@
     group="sdxl_unet_fp16",
 )
 
+# FP16 Model for 960x1024 image size
+
+sdxl_unet_fp16_960_1024_inference_input_0 = fetch_source_fixture(
+    "https://sharkpublic.blob.core.windows.net/sharkpublic/ian/unet_npys/input1.npy",
```

Comment:
nit, not blocking.

Common infra should really be pointing at a common file path with some versioning built in (like a date) and some developer documentation explaining how to manage the files. Otherwise anyone wanting to modify this in the future will need to ask you personally what to do 😛